admin/views/class-view-utils.php

```diff
@@ -108,7 +108,7 @@ public function show_post_type_settings( $post_type ) {
 			sprintf( __( '%1$s Meta Box', 'wordpress-seo' ), 'Yoast SEO' )
 		);
 
-		$editor = new WPSEO_Replacevar_Editor( $this->form, 'title-' . $post_type->name, 'metadesc-' . $post_type->name );
+		$editor = new WPSEO_Replacevar_Editor( $this->form, 'title-' . $post_type->name, 'metadesc-' . $post_type->name, false );
```

Comment:
A fourth argument for handling styling??? Should we make an issue for this labeled as `technical debt`. It would be better to have a  `WPSEO_Replacevar_Editor_Paperless` that handles this well.